### PR TITLE
Feature api endpoints

### PR DESCRIPTION
**Pull request by nakimera**
I have worked on the signup endpoint.  When a user signs up, their information is added to the users table in mydiary database

### CODE REVIEW

File: app/db.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+        except psycopg2.DatabaseError as ex:
+            self.conn.close()
+            print(ex)
+
+    def close_connection(self):
+        """
+        This method closes the database
+        """
+        self.conn.close()
```
Comment (PatrickCmd) [inline app/db.py]: Please implement instance methods in the connection class for creating and dropping your database tables

File: app/config.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+class Config:
+    SECRET_KEY = "TcQsWISFjRG4243XobHPIaDxMioisOba"
+    DEBUG = False
+    TESTING = False 
+    DEVELOPMENT = False
+
+class DevelopmentConfig(Config):
+    DEBUG = True
+    DEVELOPMENT = True
+    DATABASE_URL = 'postgresql://'+ db_user +': '+ user_password + '@localhost/'+ db_name +''
```
Comment (PatrickCmd) [inline app/config.py:14]: Please read about environment variables, I would prefer this being set as an environment variable.

**Review by PatrickCmd — COMMENTED** [review]
I see that you have not yet implemented testing, please consider writing tests for every endpoint implemented.

**Comment by nakimera** [comment]
Thank you @PatrickCmd
I have received the feedback and I am working on it

On Tue, Jul 31, 2018 at 3:57 PM, Walukagga Patrick <notifications@github.com

**Pull request merged**